### PR TITLE
[ng] router: complete base crisis center impl up to CanDeactivate

### LINKED ISSUE #375: [ng] router: add test for "save changes?" to crisis center

### PR DESCRIPTION
**Pull request by chalin**
Complete base crisis center implementation:
- The crisis center component and the crises component now have child routes.
- Added support for can deactivate router hook.

Fixes #375

### CODE REVIEW

File: src/angular/guide/router/4.jade [hunk 1]
```diff
@@ -80,16 +80,37 @@ script $('#router-milestone').val('4')
...
+//- Dart differences relative to the TS sample:
+  * `CrisisCenterComponent` has no `<h2>` element, only a router outlet. 
+    The `<h2>` is kept in the `CrisesComponent`.
+
+  TODO: mention the need for useAsDefault?
 
 :marked
+  The `CrisisCenterComponent` has a single non-terminal route handled by the 
+  `CrisesComponent`. This means that the `CrisesComponent` is a parent component
+  as well; you'll define it's child routes below.
```
Comment (kwalrath) [inline src/angular/guide/router/4.jade]: it's -> its

File: src/angular/guide/router/4.jade [hunk 2]
```diff
@@ -98,7 +119,7 @@ figure.image-display
   just as the `AppComponent` is a shell to manage the high-level workflow
 
   Like most shells, the `CrisisCenterComponent` class is very simple, simpler even than `AppComponent`:
-  it has no business logic, and it's template has no links, just a title and 
+  it has no business logic, and it's template has no links, just a
```
Comment (kwalrath) [inline src/angular/guide/router/4.jade]: it's -> its

File: src/angular/guide/router/4.jade (hunk 1 above)
Comment (chalin) [inline src/angular/guide/router/4.jade]: fixed

File: src/angular/guide/router/4.jade (hunk 2 above)
Comment (chalin) [inline src/angular/guide/router/4.jade]: fixed

**Pull request merged**